cli.js

```diff
@@ -262,7 +262,7 @@ function processDirectory(inputDir, outputDir, fileExt) {
         else if (stat.isDirectory()) {
           processDirectory(inputFile, outputFile, fileExt);
         }
-        else if (!fileExt || path.extname(file) === '.' + fileExt) {
+        else if (!fileExt || fileExt.includes(path.extname(file).substring(1))) {
```

Comment:
Please use `~fileExt.indexOf(path.extname(file).slice(1))` instead.

Reply:
Thanks. I wasn't aware `Arrays.includes` is an ES7 feature. :) Updated the pull request with an amend and a force push.